src/Components/NavBar/Menus/TwoColumnDropDownSection.tsx

```diff
@@ -0,0 +1,55 @@
+import { Box, color, Flex, MenuItem, Sans } from "@artsy/palette"
+import React from "react"
+import { MenuLinkData, SimpleLinkData } from "../menuData"
+
+interface TwoColumnDropDownSectionProps {
+  section: MenuLinkData
+}
+
+export const TwoColumnDropDownSection: React.FC<TwoColumnDropDownSectionProps> = ({
+  section,
+}) => {
+  return (
+    <Flex>
+      <Box width={[110, 110, 110, 135, 150]} py={4} mr={[2, 2, 3, 3]}>
+        <Sans size="2" mb={1}>
+          {section.text}
+        </Sans>
+        {section.menu &&
+          section.menu.links.slice(0, 8).map((menuItem: SimpleLinkData) => {
```

Comment:
So this wasn't possible to do with fixed widths/flexbox after all? Bummer!

Would you mind adding a comment to the code explaining why this logic is necessary (for our future selves 😄)?

Reply:
Would also be good to assign `section.menu.links.slice(0, 8)` to a good variable name, which is self-documenting

Reply:
That's a good call!